docs/best_practices/resilience.txt

```diff
@@ -125,9 +125,9 @@ Elasticsearch.
 Most of the known issues that relate to resiliency `exist in the Elasticsearch
 layer <https://www.elastic.co/guide/en/elasticsearch/resiliency/current/>`_.
 
-Fortunately, most of these issues are fixed in the Elasticsearch 5.x release
-line. We are working on updating CrateDB to use Elasticsearch 5.x and hope to
-have it ready in the first quarter of 2017.
+Fortunately, most of these issues are fixed in the Elasticsearch 6.x release
```

Comment:
@seut not sure if we should keep that or completely remove.

Reply:
I'd completely remove that